src/EventGrid/EventGrid/ChangeLog.md

```diff
@@ -19,6 +19,21 @@
 -->
 ## Upcoming Release
 
+## Version 1.1.1
+* Updated to use the 2019-02-01-preview API version.
+* New commands:
+    - New-AzureRmEventGridDomainTopic:
+        - Creates a new Azure Event Grid Domain Topic.
+    - Remove-AzureRmEventGridDomainTopic:
+        - Remove an existing Azure Event Grid Domain Topic.
```

Comment:
Nit: Remove should be "Removes" to be consistent with "Creates" in the previous line above.

Reply:
Fixed. thx